ledger/src/blockstore_metrics.rs

```diff
@@ -236,6 +237,9 @@ impl BlockstoreRocksDbColumnFamilyMetrics {
 // Thread local instance of RocksDB's PerfContext.
 thread_local! {static PER_THREAD_ROCKS_PERF_CONTEXT: RefCell<PerfContext> = RefCell::new(PerfContext::default());}
 
+// The minimum time interval between two RocksDB perf samples of the same operation.
+const ROCKS_PERF_METRIC_MIN_INTERVAL_MS: u64 = 1000;
```

Comment:
Thanks for the suggestion.  I like this idea. But Duration::as_millis() returns u128 instead of u64, which requires a casting when performing the comparison.  So I am still using const `u64` instead.

Reply:
`.as_millis() as u64` is  fine imo because this is a constant that we know it fits in `u64`.

Reply:
Yep, we won't have a correctness issue, but I am afraid that the compiler might not always be smart enough to cache the result of casting so I'm still using a const `1000` in `u64` instead.

But feel free to let me know if you still prefer a const `Duration` instead.